TCAT/Model/BusStop.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  BusStop.swift
+//  TCAT
+//
+//  Created by Austin Astorga on 3/26/17.
+//  Copyright © 2017 cuappdev. All rights reserved.
+//
+
+import UIKit
+import CoreLocation
+
+@objc(BusStop) class BusStop: Place, CoordinateAcceptor {
```

Comment:
This model has been deprecated. It's now bundled in `LegacyModels.swift` as we converted away from it. Should this now be removed?